architecture/additional_concepts/networking.adoc

```diff
@@ -12,3 +12,9 @@ toc::[]
 Kubernetes ensures that each pod is able to network with each other, and allocates each pod an IP address from an internal network. This ensures all containers within the pod behave as if they were on the same host. Giving each pod its own IP address means that pods can be treated like physical hosts or virtual machines in terms of port allocation, networking, naming, service discovery, load balancing, application configuration, and migration.
 
 Creating links between pods is unnecessary. However, it is not recommended that you have a pod talk to another directly by using the IP address. Instead, we recommend that you create a link:kubernetes_model.html#service[service], then interact with the service.
+
+== OpenShift SDN
+
+OpenShift deploys a software-defined networking (SDN) approach for connecting Docker containers in an OpenShift cluster. The OpenShift SDN connects all containers within a node host.
```

Comment:
~~The OpenShift SDN connects all containers within a node host.~~ The OpenShift SDN connects all containers across all node hosts.